### DISCUSSION TITLE
Unable to set the theme

### DISCUSSION THREAD

**Discussion by Raj2032** [ryanraposo/codeui#41] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29293830/165262050-35c91818-c6e5-4765-b70c-05eb5d35ef0b.png)

I have this third party theme installed but I wanted to make changes to the side bar which has the numbers on it.

Litereally any theme I attempt to change

![image](https://user-images.githubusercontent.com/29293830/165262349-064df87b-8a30-4cd2-8979-dd266b98631a.png)

When I select it

![image](https://user-images.githubusercontent.com/29293830/165262413-e187d05e-79ef-4954-b663-c99fae9beef4.png)

It does not even change

![image](https://user-images.githubusercontent.com/29293830/165262445-8bc50599-696e-417d-b46f-266426decb8f.png)

Any idea why this is happening?

**Comment by ryanraposo** [comment, marked as answer]
@Raj2032 this is being addressed (thanks to you!). I'd appreciate if you try the workaround and let me know what happens for you.

See: https://github.com/ryanraposo/codeui/issues/44

**Reply by Raj2032** [reply]
@ryanraposo Hey mate thanks for your response.  I tried you workaround and it does work which is great :)

**Reply by ryanraposo** [reply]
@Raj2032 no problem! The issue should be fixed altogether now. If you have any problems moving forward, please don't hesitate to create an Issue or ask again here! That was a pretty serious bug. When they're breaking and vague like that it can go under the radar for weeks/months unless I'm constantly testing the extension. Users tend to uninstall without feedback when _nothing_ is working in a Preview-stage extension, so I appreciate it.

Keep doin what you're doin 👍
